packages/node/src/index.ts

```diff
@@ -30,3 +30,4 @@ export * from './lib/feeds/feeds.interface';
 export * from './lib/topics/topic.interface';
 export * from './lib/integrations/integrations.interface';
 export * from './lib/messages/messages.interface';
+export { defaultRetryCondition } from './lib/retries';
```

Comment:
as I decided to make retryCondition callback configurable by user, they may want to "extend" retry condition like that:

```ts
import { Novu, defaultRetryCondition } from '@novu/node';

function myCustomRetryCondition(err) {
  if (err.response.status === 404) return true;

  return defaultRetryCondition(err);
}

const novu = new Novu('my-key', {
   retryConfig: {
     retryMax: 10,
     retryCondition: myCustomRetryCondition
   }
});
```

Reply:
Great!